pkg/kernel/Dockerfile

```diff
@@ -115,12 +115,20 @@ RUN unzip -d /tmp /tmp/xr.zip ;\
     make -C /linux INSTALL_MOD_PATH=/tmp/kernel-modules \
                    M=/tmp/xr_usb_serial_common_lnx-3.6-and-newer-pak \
          modules modules_install
+
+# Device Tree Blobs
+# FIXME: we will switch to a native make INSTALL_DTBS_PATH=/tmp/kernel-modules/boot/dtb dtbs_install at some point
```

Comment:
You already do that for new-kernel. Is there a reason it can't also be done for the kernel package?

Reply:
The reason for this is sort of embarrassing: we can no longer seem to reproduce the content of that eve.dts (which was a combination of tweaking of the Linaro/HiKey dts that no longer seems to be available).

This is, in a way, a prime example of why this PR need to go in so at least we get reproducible builds of DTBs coupled with evolving kernel features.

So my plan is to use this crutch for now (to keep HiKey going) and once we move to the new kernel we will do a clean cut-over to kernel built DTBs (and by that time I hope to mostly ditch HiKey altogether and simply focus on RPi4 as our one and only ARM development board).

Makes sense @eriknordmark ?

Reply:
OK